scripts/nginx/radarr.sh

```diff
@@ -51,18 +51,18 @@ chown -R "$user":"$user" /home/"$user"/.config/Radarr
 . /etc/swizzin/sources/functions/utils
 systemctl start radarr -q # Switch radarr on regardless whether it was off before or not as we need to have it online to trigger this cahnge
 
-sleep 5 # TODO replace with a loop until the API is available
-payload="$(curl -s "https://127.0.0.1/radarr/api/v3/config/host?apiKey=${apikey}" \
-	--user "${user}:$(_get_user_password "${user}")" --insecure \
-	-s | \
-	jq '.certificateValidation = "disabledForLocalAddresses"')"
+echo_progress_start "Waiting for Radarr API"
+if ! timeout 30 bash -c -- 'while ! curl -sfL "http://127.0.0.1:7878/api/v3/system/status?apiKey='"${apikey}"'" > /dev/null 2>&1; do sleep 5; done' ; then
+	echo_warn "There is a problem talking to the Radarr API, we need to stop here and exit."
+	exit 1
+else
+	urlbase="$(curl -sL "http://127.0.0.1:7878/api/v3/config/host?apikey=${apikey}" | jq '.urlBase' | cut -d '"' -f 2)"
+	echo_progress_done"Radarr API reachable"
+fi
+payload=$(curl -sL "http://127.0.0.1:7878/api/v3/config/host?apikey=${apikey}" | jq ".certificateValidation = \"disabledForLocalAddresses\"")
 echo_log_only "Payload = \n${payload}"
 echo_log_only "Return from radarr after PUT ="
-curl "https://127.0.0.1/radarr/api/v3/config/host?apiKey=${apikey}" -X PUT --insecure \
-	-H 'Accept: application/json, text/javascript, */*; q=0.01' \
-	--compressed -H 'Content-Type: application/x-www-form-urlencoded; charset=UTF-8' \
-	--user "${user}:$(_get_user_password "${user}")" \
-	--data-raw "$payload" -s >> "$log"
+curl -so /dev/null "http://127.0.0.1:7878${urlbase}/api/v3/config/host?apikey=${apikey}" -X PUT -H 'Accept: application/json, text/javascript, */*; q=0.01' --compressed -H 'Content-Type: application/x-www-form-urlencoded; charset=UTF-8' --data-raw "${payload}" >> "$log"
```

Comment:
`-o /dev/null` would hide the return from the log, no?

Reply:
it was to hide the output in my testing. it does not need to be there.